docs/build.md

```diff
@@ -0,0 +1,144 @@
+# vue build
+
+`vue build` command gives you a zero-configuration development setup, install once and build everywhere.
+
+## Features
+
+- **Not a boilerplate**: run a single command to develop your app
+- **Out of the box**: ES2015, single-file component with hot reloading and custom CSS preprocessors
+- **Customizable**: populate a `~/.vue/webpack.config.js` for custom webpack config
+- **Single-file component mode**: just run `vue build Component.vue` and boom!
+
+## Get started
+
+Populate a component `./component.vue` in your project:
+
+```vue
+<template>
+  <h2>Hello World!</h2>
+</template>
+```
+
+And then just run `vue build component.vue` and go to `http://localhost:4000`
+
+To build for production:
```

Comment:
What does `production` mean in this case? If I understand you'd use `vue build` to prototype or develop a single component and then include/require it in your application with its own build process. Now, it's useful to showcase minified build but maybe this can give a false signal that one can use `--prod` build in their app directly and that might lead to including vue's runtime n times in their app.

This is more of a loud thinking here :)